docker/CMSRucioClient/scripts/updateRSEs.py

```diff
@@ -0,0 +1,180 @@
+#! /bin/env python
+# Script for managing (creating/updating/deleting) the RSEs and their attributes 
+# Initially will use PhEDEx nodes information as input,
+# should be easily transformed/extended for using other sources. 
+
+import argparse
+import logging
+
+import requests
+import json
+import re
+
+from functools import wraps
+
+from rucio.client.accountclient import AccountClient
+from rucio.client.rseclient import RSEClient
+from rucio.common.exception import Duplicate, RSEProtocolPriorityError, \
+	RSEProtocolNotSupported, RSENotFound, InvalidObject
+
+DATASVC_URL='http://cmsweb.cern.ch/phedex/datasvc/json/prod/' 
+prog = re.compile('.* -service (.*?) .*')
+
+def setup_logger(logger):
+	""" Code borrowed from bin/rucio-admin
+	"""
+	logger.setLevel(logging.DEBUG)
+	hdlr = logging.StreamHandler()
+	def emit_decorator(fcn):
+		def func(*args):
+			if True:
+				formatter = logging.Formatter("%(message)s")
+			else:
+				levelno = args[0].levelno
```

Comment:
If you don't mind, please only emit the control characters when stdout is a tty.